supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -27,3 +27,28 @@ Provide the source language if it is supported by the Red Hat Customer Portal to
 == Long code examples in procedures
 
 All code blocks (regardless of length) must be necessary, accurate, and helpful. Code blocks must be as copy-and-paste friendly as possible, with the exception of variables and callouts. The length of the block is irrelevant, within reason, if the code block follows these guidelines.
+
+[[commands-with-elevated-privileges]]
+== Commands with elevated privileges
```

Comment:
@bburt-rh  @bergerhoffer @IngridT1 @ndeevy Is *elevated privileges*  the right wording here? The section only talks about `root` privileges, so why not be specific? It'll make it easier, especially for newbies, to find the information when scanning.

The term "elevated privileges" suggests that you might have more user privilege levels than just _normal_ and _root_.

Reply:
@stoobie I think it makes sense to change "elevated privileges" to "root privileges" throughout. 

Unless there are objections, I'll make the changes and update the PR.

Reply:
Good point @stoobie  "root privileges" SoundsGTM.

Reply:
Yeah, "root privileges" would be the correct nomenclature. I can see this term used in many Red Hat training docs, such as the Red Hat System Administration 1 training guide.

Reply:
@stoobie @ndeevy @dfitzmau @bergerhoffer Please review the updated content with the change from "elevated" to "root".